src/frontend/src/lib/stores/auth.store.ts

```diff
@@ -85,6 +89,14 @@ const initAuthStore = (): AuthStore => {
 				...state,
 				identity: null
 			}));
+		},
+
+		setForTesting: (identity) => {
+			if (!LOCAL && !TEST) {
```

Comment:
Honest curiosity, since I am not so expert of `vitest`: why `LOCAL` is allowed? Isn't it just for testing? Doesn't work on local replica when you run `npm run test` ?

Reply:
Good point. Let's just use TEST.

Reply:
Done.